browser/ui/webui/settings/brave_email_aliases_handler.cc

```diff
@@ -0,0 +1,447 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/webui/settings/brave_email_aliases_handler.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/functional/bind.h"
+#include "base/functional/callback.h"
+#include "base/json/json_reader.h"
+#include "base/json/json_writer.h"
+#include "base/values.h"
+#include "brave/browser/brave_browser_process.h"
+#include "brave/browser/ui/webui/email_aliases/email_aliases_bubble_ui.h"
+#include "brave/components/constants/webui_url_constants.h"
+#include "brave/components/email_aliases/browser/pref_names.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/singleton_tabs.h"
+#include "components/prefs/pref_service.h"
+#include "components/prefs/scoped_user_pref_update.h"
+#include "content/public/browser/web_ui.h"
+#include "net/http/http_request_headers.h"
+#include "net/traffic_annotation/network_traffic_annotation.h"
+#include "services/network/public/cpp/resource_request.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
+#include "url/gurl.h"
+
+#define MAX_RESPONSE_LENGTH 32768
+
+const char kAccountsServiceRequestURL[] =
+    "https://accounts.bsg.bravesoftware.com/v2/verify/init";
+const char kAccountsServiceVerifyURL[] =
+    "https://accounts.bsg.bravesoftware.com/v2/verify/result";
+const char kMappingServiceManageURL[] =
+    "https://aliases.bsg.bravesoftware.com/manage";
+const char kMappingServiceGenerateURL[] =
+    "https://aliases.bsg.bravesoftware.com/generate";
+const char kBraveApiKey[] = "px6zQ7rIMGaS8FE6cmpUp45WQTFJYXgo7ZlBhrFK";
```

Comment:
this api key needs to be removed from the code and the server now since it has been published on github